modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase

```diff
@@ -194,7 +194,9 @@ definition {
 			structureName = "WC Structure Name",
 			webContentTitle = "WebContent Title");
 
-		Click(locator1 = "Button#PUBLISH");
+		Click(
+			key_text = "Publish",
+			locator1 = "Button#ANY");
```

Comment:
The publish button has changed.
The tests have been tried and the result is in the ticket.